### PR TITLE
Supporting duplicateses on multimap

### PR DESCRIPTION
**Pull request by miador**
Closes [298](https://github.com/infinispan/infinispan.github.io/issues/298)

### CODE REVIEW

**Review by karesti — APPROVED** [review]

**Comment by karesti** [comment]
@dvagnero any thoughts?

File: _posts/2022-10-02-infinispan-14-supporting-duplicates-on-multimap.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+date: '2022-10-2T00:00:00.000-00:00'
+author: miador
+tags: [ "multimap" ]
+---
+
+= Infinispan 14 supporting duplicates on multimap
+
+Dear Infinispan community,
+
+with the Infinispan 14 development release 04 multimap can support duplicates.
```
Comment (dvagnero) [inline _posts/2022-10-02-infinispan-14-supporting-duplicates-on-multimap.adoc:14]:
```suggestion
With the Infinispan 14 development release 04 multimap cache supports duplicates.
```
Just a suggestion @miador I would add one more sentence about the benefits that this brings. Why should I use it?

File: _posts/2022-10-02-infinispan-14-supporting-duplicates-on-multimap.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+By default `supportsDuplicates` is set to false and can be configured during initialization.
+
+Here is an about how can we set multimap to support duplicate key-value pairs:
```
Comment (dvagnero) [inline _posts/2022-10-02-infinispan-14-supporting-duplicates-on-multimap.adoc:18]:
```suggestion
The following is an example on how you can set multimap to support duplicate key-value pairs:
```

**Comment by dvagnero** [comment]
Hi @karesti and @miador, just a small suggestion. 
And possibly an item for docs as it seems that the docs are outdated https://infinispan.org/docs/14.0.x/titles/embedding/embedding.html#multimap_limitations

**Comment by miador** [comment]
Hi @dvagnero, thanks for suggestions. I'll update them tomorrow. And yes, i agree the docs are outdated. I can update them in Infinispan repo. Should i open a PR for that?

**Comment by karesti** [comment]
@miador yes to an update docs PR please :)

**Review by karesti — APPROVED** [review]

**Comment by karesti** [comment]
@miador we can squash the commits now

**Review by dvagnero — APPROVED** [review]
LGTM :)

**Pull request merged**